confignode/src/main/java/org/apache/iotdb/confignode/consensus/response/DataPartitionResp.java

```diff
@@ -49,28 +61,51 @@ public boolean isAllPartitionsExist() {
     return allPartitionsExist;
   }
 
-  public void setAllPartitionsExist(boolean allPartitionsExist) {
-    this.allPartitionsExist = allPartitionsExist;
-  }
-
-  public void setDataPartition(DataPartition dataPartition) {
-    this.dataPartition = dataPartition;
-  }
-
-  public DataPartition getDataPartition() {
-    return dataPartition;
-  }
-
   /**
    * Convert DataPartitionDataSet to TDataPartitionResp
    *
    * @param resp TDataPartitionResp
    */
-  public void convertToRpcDataPartitionResp(TDataPartitionResp resp) {
+  public void convertToRpcDataPartitionResp(
```

Comment:
Suggest make the `resp` as return value rather than a parameter, which makes this method not pure.

Reply:
Fixed.